Makefile

```diff
@@ -84,41 +80,57 @@ TEMP_DIR := $(shell mktemp -d)
 all: all-container
 
 sub-container-%:
-	$(MAKE) --no-print-directory ARCH=$* container
+	# 'clean' is necessary since build process needs to overwrite kube-state-metrics binary. Otherwise all images end up with binary made for one architecture
```

Comment:
Why not have `clean` as a pre-requisite to  the `sub-container-%` target?

Reply:
That was the first thing I did, but it didn't work out :shrug: